admin/scripts/common/models/__init__.py

```diff
@@ -0,0 +1,105 @@
+from enum import Enum
+from typing import Any, Optional
+
+from pydantic import BaseModel, Field, field_validator, model_validator
+
+
+class Choice(str, Enum):
+    yes = "Yes"
+    not_planned = "Not planned"
+    not_yet = "Not yet"
+
+
+class Choice2(str, Enum):
+    yes = "Yes"
+    not_sure_yet = "Not sure yet"
+    no = "No"
+
+
+class ChoiceReminder(str, Enum):
+    no = "No"
+    yes_one_week = "Yes, in one week"
+    yes_one_month = "Yes, in one month"
+
+
+class ProjectRole(str, Enum):
+    principal_author_lead_engineer = "Principal author / lead engineer"
+    core_contributor_maintainer = "Core contributor / maintainer"
+    release_manager = "Release manager"
+    security = "Security"
+    infrastructure_devops = "Infrastructure/DevOps"
+    community_manager = "Community manager"
+    outreach_coordinator = "Outreach coordinator"
+    other = "Other"
+
+
+class UpdateFrequency(str, Enum):
+    cant_say = "Can't say"
+    more_once_per_day = "More than once per day"
+    more_once_per_week = "More than once per week"
+    more_once_per_month = "More than once per month"
+    more_once_per_year = "More than once per year"
+    less_once_per_year = "Less than once per year"
+
+
+question_alias_mapping = {
+    "q1": "project_name",
+    "q2": "author_role",
+    "q3": "build_failure_duration",
```

Comment:
There is no uniform formatting for this question. Some responses provide the number of hours per week directly (which we can validate and format as we want) while other go into more details.

Does it make sense to restrict the response field to numbers only?

Reply:
Let's keep it a string for now, this is a qualitative metric we need at the moment for prioritisation, but we likely won't have to work with it programmatically. Unless you think we do, then make it a number in a suitable unit, and we'll clean the data accordingly.